### PR TITLE
update nuget

### PR DESCRIPTION
**Pull request by BelehovEgor**

### CODE REVIEW

File: src/PositionService.Validation/PositionUser/CreatePositionUserRequestValidator.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      IPositionUserRepository positionUserRepository,
+      IPositionRepository positionRepository)
+    {
+      RuleFor(x => x)
+        .MustAsync(async (request, _) =>
+        {
+          var position = await positionUserRepository.GetAsync(request.UserId);
+          return position is not null && position.IsActive && position.PositionId != request.PositionId;
+        })
+        .WithMessage("Incorrect position for this user");
```
Comment (MariyaSin) [inline src/PositionService.Validation/PositionUser/CreatePositionUserRequestValidator.cs:20]: add '.' at the end of the sentence

File: src/PositionService.Broker/Consumers/CreateUserPositionConsumer.cs [hunk 2]
```diff
@@ -21,17 +22,25 @@ private async Task<bool> CreateAsync(ICreateUserPositionRequest request)
         return false;
       }
 
-      return (await _userRepository.CreateAsync(_mapper.Map(request))) != null;
+      await Task.WhenAll(
+        _userRepository.CreateAsync(_positionMapper.Map(request)),
+        _userRateRepository.CreateAsync(_rateMapper.Map(request)));
+
+      return true;
```
Comment (MariyaSin) [inline src/PositionService.Broker/Consumers/CreateUserPositionConsumer.cs:29]: remove something

**Review by MariyaSin — APPROVED** [review]

**Review by Cheephot — APPROVED** [review]

File: src/PositionService.Mappers/Db/DbPositionUserMapper.cs [hunk 3]
```diff
@@ -19,11 +29,28 @@ public DbPositionUser Map(ICreateUserPositionRequest request)
...
-        Rate = request.Rate,
         IsActive = true,
         CreatedBy = request.CreatedBy,
         CreatedAtUtc = DateTime.UtcNow
       };
     }
+
+    public DbPositionUser Map(CreatePositionUserRequest request)
+    {
+      if (request == null)
```
Comment (VLiamin) [inline src/PositionService.Mappers/Db/DbPositionUserMapper.cs:40]: Why do we write in some places "== null" in others "is null"?

**Review by VLiamin — APPROVED** [review]

File: src/PositionService.Mappers/Db/DbPositionUserMapper.cs (hunk 3 above)
Comment (BelehovEgor) [reply src/PositionService.Mappers/Db/DbPositionUserMapper.cs]: fixed

File: src/PositionService.Validation/PositionUser/CreatePositionUserRequestValidator.cs (hunk 1 above)
Comment (BelehovEgor) [reply src/PositionService.Validation/PositionUser/CreatePositionUserRequestValidator.cs]: fixed

**Review by MariyaSin — APPROVED** [review]

**Review by asyagraf — APPROVED** [review]

**Review by ndoriomedov — APPROVED** [review]

**Pull request merged**